src/Routes.js

```diff
@@ -23,7 +23,7 @@ const Routes = () => (
     }
   >
     <Switch>
```

Comment:
The `Switch` is no longer officially supported. This code depends on a compatibility layer.

It was replaced by the `Routes` component.